JL.Core/Mining/MiningUtils.cs

```diff
@@ -74,6 +74,127 @@ public static class MiningUtils
         </style>
         """;
 
+    private static string? GetMiningParameter(JLField field, LookupResult lookupResult, string currentText, int currentCharPosition, bool useHtmlTags)
```

Comment:
```suggestion
    private static string? GetMiningParameter(JLField field, LookupResult lookupResult, string currentText, int currentCharPosition)
```

`useHtmlTags` will always be true for this method